tests/sharp_interfaces_06.cc

```diff
@@ -124,15 +124,18 @@ MicroFluidicProblem<dim>::run()
   navier_stokes_solver.setup_problem(Functions::ZeroFunction<dim>(dim));
   navier_stokes_solver.output_solution(parameters.output_filename);
 
-  Triangulation<dim - 1, dim> surface_mesh;
+  parallel::shared::Triangulation<dim - 1, dim> surface_mesh(mpi_communicator);
```

Comment:
Does this mean that we need to ensure deal.II is built with METIS? Or can we make the test optional? I would like to avoid a required dependency on too many external libraries (unless we can provide a spack recipe or something similar that helps the user along).

Reply:
As far as I know, if now METIS or Zoltan are available the class will use partitioning along a z-curve so it should work.

Reply:
Ah ok, that would be great. So let's keep it like this (I won't test it immediately because I have metis on my installations.)